clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -41,3 +41,18 @@
       In: ${arg}.cu
       Out: ${arg}.dp.cpp
 
+- Rule: rule_cuda_compile_cubin
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: cuda_compile_cubin
+  MatchMode: Partial
+  In: cuda_compile_cubin(${value})
```

Comment:
This line does performs compiling, from a.cu to a.cubin.
So, in SYCL side. the mapping should also perform compiling, from a.dp.cpp to a.dp.o with "-c" option.